.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5500
```

Comment:
Shall we add VSCode to the list of technologies to include in our gitignore?

Reply:
(Remember the whole `git rm --cached` thing)

Reply:
I think I've done this now. :)